src/public/modules/forms/base/directives/submit-form.directive.js

```diff
@@ -321,7 +321,7 @@ function submitFormDirective(
             responseMode: form.responseMode,
           },
           submissionContent, // POST body
-        ).then(handleSubmitSuccess, handleSubmitFailure)
+        ).then((response) => handleSubmitSuccess(response), handleSubmitFailure)
```

Comment:
would it be better to omit the parameter for consistency? (or add the error to `handleSubmitFailure` also)

Reply:
The parameter was added as the `handleSubmitSuccess` function now has to extract the response ID from the response. I can add the error parameter the `handleSubmitFailure`.

Reply:
i think how it works is the parameters get passed implicitly actually! but yea, if it adds clarity then should add the error parameter to `handleSubmitFailure`